test/pom.xml

```diff
@@ -224,6 +224,31 @@ THE SOFTWARE.
         <!-- version specified in grandparent pom -->
         <extensions>true</extensions>
       </plugin>
+      <plugin>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>old-remoting-for-test</id>
+            <phase>generate-test-resources</phase>
+            <goals>
+              <!-- we use copy as this is a dependency from outside the reactor -->
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.jenkins-ci.main</groupId>
+                  <artifactId>remoting</artifactId>
+                  <version>${remoting.minimum.supported.version}</version>
+                  <type>jar</type>
+                  <outputDirectory>${project.build.outputDirectory}/old-remoting</outputDirectory>
+                  <destFileName>remoting-minimal-supported.jar</destFileName>
```

Comment:
BTW may want to s/minimal/minimum/ here too for consistency.

Reply:
Will address it in a follow-up